ts/parts/SvgRenderer.ts

```diff
@@ -959,26 +960,31 @@ extend((
     animate: function (
         this: Highcharts.SVGElement,
         params: Highcharts.SVGAttributes,
-        options?: (boolean|Highcharts.AnimationOptionsObject),
+        options?: (boolean|Partial<Highcharts.AnimationOptionsObject>),
         complete?: Function
     ): Highcharts.SVGElement {
         var animOptions = animObject(
-            pick(options, this.renderer.globalAnimation, true)
-        );
+                pick(options, this.renderer.globalAnimation, true)
+            ),
+            deferTime = animOptions.defer;
+
 
         // When the page is hidden save resources in the background by not
         // running animation at all (#9749).
         if (pick(doc.hidden, doc.msHidden, doc.webkitHidden, false)) {
             animOptions.duration = 0;
         }
 
-        if (animOptions.duration !== 0) {
+        if (animOptions.duration !== 0 && defined(deferTime)) {
```

Comment:
Is that necessary? Doesn't now `animObject`always return `{ duration, defer }` ?